packages/create-waku-app/createApp.js

```diff
@@ -0,0 +1,121 @@
+const path = require("path");
+const fs = require("fs-extra");
+const execSync = require("child_process").execSync;
+
+const { Command } = require("commander");
+const validateProjectName = require("validate-npm-package-name");
+
+const DEFAULT_TEMPLATE = "web-chat";
+const supportedExamplesDir = path.resolve(__dirname, "./examples");
+
+const init = (name, description, version, supportedExamples) => {
+    let appName;
+    const program = new Command()
+        .name(name)
+        .description(description)
+        .version(version, "-v, --version", "output the version number")
+        .arguments("<project-directory>", "Project directory to initialize Waku app")
+        .action(_appName => {
+            appName = _appName;
+        })
+        .option(
+            "-t, --template <path-to-template>", 
+            "specify a template for the created project"
+        )
+        .allowUnknownOption()
+        .parse();
```

Comment:
If I understand this correctly (haven't confirmed yet as I couldn't run it locally), the CLI expects options to be passed within the `yarn create xyz` command -- could we maybe change it to having the CLI output the options and enable user to choose?

Similar to how Hardhat does it - for reference: (https://hardhat.org/hardhat-runner/docs/getting-started#overview)
```$ npx hardhat
👷 Welcome to Hardhat v2.9.9 👷‍

? What do you want to do? …
❯ Create a JavaScript project
  Create a TypeScript project
  Create an empty hardhat.config.js
  Quit
```

Reply:
This is a good idea, we can have a follow-up task to enable this but I would add it as additional feature / separate package. Sometimes it is hard to make working in different shells.